components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -85,6 +85,21 @@ func (p *pg) CreatePolicy(ctx context.Context, pol *v2.Policy) (*v2.Policy, erro
 		return nil, p.processError(err)
 	}
 
+	if checkProjects {
+		err = p.errIfMissingProjectsWithQuerier(ctx, tx, pol.Projects)
+		if err != nil {
+			return nil, p.processError(err)
+		}
+
+		err = projectassignment.ErrIfProjectAssignmentUnauthorized(ctx,
```

Comment:
Naming is hard -- but can we invert the _naming_ logic here? I.e. don't say "err if yadda", but "ensure yadda" or something like that, and if it returns an error, that obviously has failed.